src/themes/blueberry/Style.js

```diff
@@ -43,7 +43,6 @@ export default () => (
         border: solid 2px #EDEDED;
         background: #fff;
         
-        display: block;
```

Comment:
you're changing the `blueberry` theme, but I think yo meant to change `greyVest`

Reply:
Good point. Fixed